src/lib/server.ts

```diff
@@ -233,6 +238,36 @@ export class ClientsController {
       this.eventForwarder = new EventForwarder(this);
       this.eventForwarder.start();
     }
+
+    if (this.loggingEventForwarder === undefined) {
+      this.loggingEventForwarder = new LoggingEventForwarder(this, this.driver);
+    }
+  }
+
+  get loggingEventForwarderStarted(): boolean {
+    if (
+      !this.loggingEventForwarder ||
+      this.loggingEventForwarder === undefined
+    ) {
+      return false;
+    }
+    return this.loggingEventForwarder.started;
+  }
+
+  public configureLoggingEventForwarder() {
+    if (!this.loggingEventForwarderStarted) {
+      this.loggingEventForwarder?.start();
+    }
+  }
+
+  public cleanupLoggingEventForwarder() {
+    if (
+      this.clients.filter((cl) => cl.receiveLogs && cl.isConnected).length ==
```

Comment:
We don't care about connected here I think. 
```suggestion
      this.clients.filter((cl) => cl.receiveLogs).length ==
```